src/ops.rs

```diff
@@ -169,6 +176,38 @@ pub fn sahf(vm: &mut VM, _pipeline: &Pipeline, _hv: &mut dyn Hypervisor) -> Resu
     Ok(())
 }
 
+pub fn stc(vm: &mut VM, _pipeline: &Pipeline, _hv: &mut dyn Hypervisor) -> Result<(), VMError>{
+    vm.flags.carry = true;
+    Ok(())
+}
+
+pub fn clc(vm: &mut VM, _pipeline: &Pipeline, _hv: &mut dyn Hypervisor) -> Result<(), VMError>{
+    vm.flags.carry = false;
+    Ok(())
+}
+
+pub fn cmc(vm: &mut VM, _pipeline: &Pipeline, _hv: &mut dyn Hypervisor) -> Result<(), VMError>{
+    vm.flags.carry = !vm.flags.carry;
+    Ok(())
+}
+
+pub fn xlatb(vm: &mut VM, pipeline: &Pipeline, _hv: &mut dyn Hypervisor) -> Result<(), VMError>{
+    if pipeline.size_override {
+        let al = vm.get_reg(Reg8::AL as u8, ValueSize::Byte).u16_zx()?;
+        let bx = vm.get_reg(Reg16::BX as u8, ValueSize::Word).u16_exact()?;
+        let sum = al.wrapping_add(bx);
+        let result = vm.get_mem(sum as u32, ValueSize::Byte)?.u8_exact()?;
+        vm.set_reg(Reg8::AL as u8, SizedValue::Byte(result));
+    } else {
+        let al = vm.get_reg(Reg8::AL as u8, ValueSize::Byte).u32_zx()?;
+        let bx = vm.get_reg(Reg32::EBX as u8, ValueSize::Dword).u32_exact()?;
```

Comment:
although it doesn't change logic, in order to make the code a bit clearer, the `bx` variable should be renamed to `ebx`

Reply:
done